wicket-ioc/src/main/java/org/apache/wicket/proxy/LazyInitProxyFactory.java

```diff
@@ -349,6 +368,33 @@ public Object writeReplace() throws ObjectStreamException
 	}
 
 	/**
+	 * Serializable implementation of the NoOp callback.
+	 */
+	public static class SerializableNoOpCallback implements NoOp, Serializable
+	{
+		private static final long serialVersionUID = 1L;
+	}
+
+	/**
+	 * CGLib callback filter: does not intercept protected methods.
+	 * 
+	 * Otherwise CGLib throws an IllegalArgumentException: protected method.
+	 */
+	private static class CGLibCallbackFilter implements CallbackFilter {
```

Comment:
Please add a link to the description of the problem with protected methods in the Javadoc.